plugins/copr.py

```diff
@@ -463,21 +464,52 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
 
         for line in f:
             if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
+                self.repo_filename = os.path.join(self.base.conf.get_reposdir,
+                                                  "_" + line[1:-2] + ".repo")
             break
 
         # if using default hub, remove possible old repofile
         if self.copr_url == self.default_url:
             # copr:hub:user:project.repo => _copr_user_project.repo
-            old_repo_filename = repo_filename.replace("_copr:", "_copr", 1)\
+            old_repo_filename = self.repo_filename.replace("_copr:", "_copr", 1)\
                 .replace(self.copr_hostname, "").replace(":", "_", 1).replace(":", "-")\
                 .replace("group_", "@")
             if os.path.exists(old_repo_filename):
                 os.remove(old_repo_filename)
 
-        shutil.copy2(f.name, repo_filename)
-        os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+        shutil.copy2(f.name, self.repo_filename)
+        os.chmod(self.repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+
+    def _runtime_deps_warning(self, project_name, chroot=None):
+        repo_file = ConfigParser(strict=False)
+        repo_file.read(self.repo_filename)
+
+        runtime_deps = []
+        for repo in repo_file.sections():
+            if repo.startswith("copr:"):
+                continue
+            runtime_deps.append(repo)
+
+        if not runtime_deps:
+            return
+
+        print(_("The Copr project {0} uses the following repositories as runtime dependencies:"
+              .format(project_name)))
+        dep_idx = 1
+        for repo in runtime_deps:
+            print("{0}. [{1}]".format(dep_idx, repo))
+            print("   baseurl={0}".format(repo_file.get(repo, "baseurl")))
+            dep_idx += 1
+
+        msg = _("Do you want to keep them enabled?")
+        if self.base._promptWanted():
+            if self.base.conf.assumeno or not self.base.output.userconfirm(
+                    msg='{} [y/N]: '.format(msg), defaultyes_msg='{} [Y/n]: '.format(msg)):
+                for dep in runtime_deps:
+                    repo_file.set(dep, "enabled", "0")
+
+                with open(self.repo_filename, "w") as f:
+                    repo_file.write(f)
```

Comment:
What if we used the same mechanism as we use for `disable`?  (Instead of writing the file from scratch)